src/main/java/io/vertx/core/eventbus/impl/clustered/ClusterNodeInfo.java

```diff
@@ -0,0 +1,43 @@
+package io.vertx.core.eventbus.impl.clustered;
```

Comment:
Sorry, I missed that, could you a copyright section at the top of the file? Thanks